app/assets/javascripts/scribing_canvas.js

```diff
@@ -82,7 +113,7 @@ function loadScribble(c, scribble, layersList) {
       .attr('selected','selected');
     layersList.append(newLayerEntry);
 
-    var toggleLayer = function (showLayer) {
+    function toggleLayer(showLayer) {
```

Comment:
Function declarations should not be placed in blocks. Use a function expression or move the statement to the top of the outer function.